packages/components/src/local/form-elements/rcb/index.tsx

```diff
@@ -16,10 +16,7 @@ const buildStyles = (options: any[]): any => {
   const opts = {}
   options.forEach((option, idx) => {
     opts[`root-${idx}`] = {}
```

Comment:
well, I wasn't expecting it to be this simple 🤣 

Does the UI for this form still look ok @lebaphi  ?

Reply:
Yep, it works fine @IanMayo but I see an error when loading the page on first time.

https://user-images.githubusercontent.com/26705621/176198455-3d4f1511-f731-4a8e-a916-ab086235de38.mov

Reply:
Ok, I see a few errors:

* `Warning: Cannot update a component` - yes, it would be good to fix this.  I'll create an issue for it.
* `[Violation] 'message' handler` - I don't know much about this
* `HEAD http://localhost:3000/assets/couners/unk` - Yes, this may be an issue.  I'll investigate.